src/Book.ts

```diff
@@ -2,5 +2,6 @@
 class Book {
   prev: Book;
   next: Book;
+  chCount: number;
```

Comment:
Please use the full name. We aren't hurting for bytes on disk

Reply:
Thanks to UCSD, we now have exabytes to work with!